dashboards/templates/dashboards/components/filter.html

```diff
@@ -0,0 +1,9 @@
+<form id="filter-form" hx-post="{% url 'your_dashboard_refresh_url' %}" hx-trigger="submit">
```

Comment:
This should not be an example.  It needs to be a working template that knows how to display the django filter form e.g.

```
<form method="get">
        {{ rendered_value.filter.form.as_p }}
        <input type="submit" />
    </form>
    ```
    You will need more than this

Reply:
Sure, I will create a working template also looking into dashboard. Html code to render it over there